### ISSUE TITLE
appengine deploy doesn't like api_version: go1.4.1

### ISSUE THREAD

**Issue by tryggth** [cayleygraph/cayley#332]
The log is:

L-SFOD8AG3QC-M:cayley ja3r9v3$ goapp deploy -application bold-streamer-111423 appengine/
04:26 PM Application: bold-streamer-111423; version: 1
04:26 PM Host: appengine.google.com
****************************************************************
The api version you are using (go1.4.1) is obsolete!  You should
upgrade your SDK and test that your code works with the new
api version.
-----------
Latest SDK:
api_versions: ['1']
release: 1.9.28
supported_api_versions:
  go:
    api_versions: [go1]
  java7:
    api_versions: ['1.0']
  python:
    api_versions: ['1']
  python27:
    api_versions: ['1']
timestamp: 1442522553

-----------
Your SDK:
api_versions: ['1']
release: 1.9.27
supported_api_versions:
  go:
    api_versions: [go1]
  java7:
    api_versions: ['1.0']
  python:
    api_versions: ['1']
  python27:
    api_versions: ['1']
timestamp: 1439922504

-----------
Please visit https://developers.google.com/appengine/downloads
for the latest SDK
****************************************************************
04:26 PM 
Starting update of app: bold-streamer-111423, version: 1
04:26 PM Getting current resource limits.
04:26 PM Scanning files on local disk.
04:26 PM Cloning 145 application files.
04:26 PM Uploading 1 files and blobs.
04:26 PM Uploaded 1 files and blobs.
04:26 PM Compilation starting.
04:26 PM Compilation: 143 files left.
04:26 PM Error 422: --- begin server output ---
Unknown api_version: go1.4.1
--- end server output ---
04:26 PM Rolling back the update.
Error 422: --- begin server output ---

--- end server output ---
error while running appcfg.py: exit status 1

Changing to: go1 gets past this error.
Will submit a new issue for subsequent issue.

**Comment by tryggth** [comment]
When the api_version is set to go1, the running app returns the following within the browser:
----------------------------------------------------------------------------------------------------
Error: Server Error

The server encountered an error and could not complete your request.
Please try again in 30 seconds.
---------------------------------------------------------------------------------------------------

The monitoring log shows:

panic: html/template: pattern matches no files: `../templates/*.tmpl`

goroutine 1 [running]:
html/template.Must(0x0, 0x7f6527af2d00, 0xc010420f30, 0x7f6527af2d00)
	go/src/html/template/template.go:304 +0x65
github.com/google/cayley/internal/http.SetupRoutes(0xc010455560, 0xc01042f780)
	github.com/google/cayley/internal/http/http.go:147 +0x3ec
main77922.init·1()
	appengine.go:111 +0x2e0
main77922.init()
	_extra_imports_29.go:7 +0xaa
main.init()
	_go_main.go:28 +0x91

goroutine 5 [chan receive]:
runtime.gopark(0xafe8e0, 0xc010424778, 0x138d490, 0xc)
	/tmp/appengine/go/src/runtime/proc.go:130 +0x11e
runtime.goparkunlock(0xc010424778, 0x138d490, 0xc)
	/tmp/appengine/go/src/runtime/proc.go:136 +0x5d
runtime.chanrecv(0x10fbe80, 0xc010424720, 0xc010418fb0, 0x1, 0x0)
	/tmp/appengine/go/src/runtime/chan.go:467 +0x959
runtime.chanrecv2(0x10fbe80, 0xc010424720, 0xc010418fb0, 0x0)
	/tmp/appengine/go/src/runtime/chan.go:316 +0x2b
_(0x185d5c0)
	_:989 +0x8d
runtime.goexit()
	/tmp/appengine/go/src/runtime/asm_amd64.s:2246 +0x1

goroutine 6 [IO wait]:
internal/net.(*pollDesc).Wait(0xc010410290, 0x72, 0x0, 0x0)
	go/src/internal/net/fd_poll_runtime.go:84 +0x5c
internal/net.(*pollDesc).WaitRead(0xc010410290, 0x0, 0x0)
	go/src/internal/net/fd_poll_runtime.go:89 +0x58
internal/net.(*netFD).readMsg(0xc010410230, 0xc01055e000, 0x10000, 0x10000, 0xc010427310, 0x50, 0x50, 0xffffffffffffffff, 0x0, 0x0, ...)
	go/src/internal/net/fd_unix.go:296 +0x5a6
internal/net.(*UnixConn).ReadMsgUnix(0xc01042c0b8, 0xc01055e000, 0x10000, 0x10000, 0xc010427310, 0x50, 0x50, 0x50, 0x0, 0x0, ...)
	go/src/internal/net/unixsock_posix.go:147 +0x1ad
runtime/eventloop.(*UnixConn).ReadMsgUnix(0xc01042c0c0, 0xc01055e000, 0x10000, 0x10000, 0xc010427310, 0x50, 0x50, 0xc0104c27c8, 0x1, 0x0, ...)
	go/src/runtime/eventloop/fd.go:60 +0xa6
created by appengine_internal.InitUDRPC
	go/src/appengine_internal/api_prod.go:76 +0x104

goroutine 7 [chan receive]:
github.com/barakmich/glog.(*loggingT).flushDaemon(0x185d900)
	github.com/barakmich/glog/glog.go:923 +0x8d
created by github.com/barakmich/glog.init·1
	github.com/barakmich/glog/glog.go:408 +0x375

goroutine 9 [chan receive]:
appengine_internal.(*server).HandleRequest(0x185b980, 0xc0104019e0, 0xc0104f6000, 0xc010423380, 0x0, 0x0)
	go/src/appengine_internal/api_prod.go:141 +0x8c
reflect.Value.call(0x128daa0, 0x185b980, 0x113, 0x13488b0, 0x4, 0xc01053df78, 0x3, 0x3, 0x0, 0x0, ...)
	/tmp/appengine/go/src/reflect/value.go:419 +0x10fd
reflect.Value.Call(0x128daa0, 0x185b980, 0x113, 0xc01053df78, 0x3, 0x3, 0x0, 0x0, 0x0)
	/tmp/appengine/go/src/reflect/value.go:296 +0xe3

**Comment by tryggth** [comment]
Actually, that was after I removed a couple of source files for REPL. Before that I got a compile error:

Starting update of app: bold-streamer-111423, version: 1
04:44 PM Getting current resource limits.
04:44 PM Scanning files on local disk.
04:44 PM Cloning 159 application files.
04:44 PM Uploading 1 files and blobs.
04:44 PM Uploaded 1 files and blobs.
04:44 PM Compilation starting.
04:44 PM Compilation: 157 files left.
04:44 PM Error 422: --- begin server output ---
Compile failed:
2015/10/29 16:44:38 go-app-builder: Failed parsing input: parser: bad import "syscall" in github.com/peterh/liner/input.go
--- end server output ---
04:44 PM Rolling back the update.
Error 422: --- begin server output ---

--- end server output ---
error while running appcfg.py: exit status 1
